fury/actor.py

```diff
@@ -1609,21 +1609,93 @@ def cylinder(centers, directions, colors, radius=0.05, heights=1,
     >>> # window.show(scene)
 
     """
-    src = CylinderSource() if faces is None else None
-
-    if src is not None:
```

Comment:
You need to keep this. It allows using your own faces and vertices if you build yourself a cylinder.

it give us flexibility for future change in the API

Reply:
This appears to be from an early commit, before I addressed Guaje's comments. In the more recent commits, values for src and rotate are appropriately set to None if faces is not None.

Reply:
Can you point which commit? 

I do not find it on master as you can see [here](https://github.com/fury-gl/fury/blob/master/fury/actor.py#L1612)

Reply:
Also, can you revert it? thank you

Reply:
I’m looking on the glyph_orientation branch of my GitHub, [`here`](https://github.com/Sassafrass6/fury/blob/glyph_orientation/fury/actor.py#L1612). 

The link you’ve posted is from fury_gl:master, right?
There is no orientation argument, no disk function, etc. 

What are you referencing, when you ask me to “revert it”?

Reply:
Yes, this is `fury-gl:master`. If I merge your PR as it is, this modification will be applied which we do not want. 

So either:
- you rebase your PR
- you merge `master` in your branch.
- you just change this part of the code yourself and commit it.

Sorry for not being explicit, this is what I meant by reverting it (keep this feature already in master).

Reply:
Ok, I review the history of this, no need to do any changes, I will go ahead and merge it. Thanks @Sassafrass6

Reply:
I see what you mean now. 
15 commits is a lot to review. Thank you for looking through it all.